openml/extensions/sklearn/__init__.py

```diff
@@ -7,3 +7,19 @@
 __all__ = ["SklearnExtension"]
 
 register_extension(SklearnExtension)
+
+
+def cont(X):
+    """Returns True for all non-categorical columns, False for the rest.
```

Comment:
Could you please extend the docstring to say why this function exists?

Reply:
Is it adequate now?

Reply:
Yes, it is!